transitionmonitor_docker/Dockerfile

```diff
@@ -1,23 +1,133 @@
-FROM 2dii/r-packages@sha256:2e99aaf36141cade6bc6f51023dfdf6dba87c464ee17a062207a37707553d69a
+# using R 4.1.2, and CTAN and CRAN packages available as of 2021.11.18
+# https://hub.docker.com/_/r-base
+# https://www.texlive.info/tlnet-archive
+# https://packagemanager.rstudio.com/client/#/repos/1/overview
 
-RUN echo "options(tinytex.tlmgr.path = '/root/.TinyTeX/bin/x86_64-linux/tlmgr')" > .Rprofile \
+FROM r-base:4.1.2
+
+# install system dependencies
+ARG SYS_DEPS="\
+    git \
```

Comment:
does `git` really need to be installed inside the container?

Reply:
ideally, yes...

1. in our/my current process, we tag the repos from within the docker container (which doesn't necessarily have to be the case, but we need a robust alternative)
2. the environment_info.json export includes the local git tag and git hash (this is for a backup/comparison to the environment variable `build_version`, so it's not totally necessary, but I thought it was a good backup)